packages/common/src/groups/_internal/GroupBusinessRules.ts

```diff
@@ -248,13 +248,13 @@ export const GroupPermissionPolicies: IPermissionPolicy[] = [
     dependencies: ["hub:group:workspace"],
   },
   {
-    permission: "hub:group:workspace:discussion",
-    services: ["discussions"],
+    permission: "hub:group:workspace:settings",
     dependencies: ["hub:group:workspace"],
   },
   {
-    permission: "hub:group:workspace:settings",
-    dependencies: ["hub:group:workspace"],
+    permission: "hub:group:settings:discussions",
+    services: ["discussions"],
+    dependencies: ["hub:group:workspace:settings"],
```

Comment:
we should add `licenses: ["hub-basic", "hub-premium"]` to this permission policy definition, else the `Settings => Discussions` workspace pane will render in the group workspace in enterprise-sites (similar to [14010](https://devtopia.esri.com/dc/hub/issues/14010) & [PR-1984](https://github.com/Esri/hub.js/pull/1984)).